dump/dump.js

```diff
@@ -101,34 +111,40 @@ const options = commandLineArgs(optionsDefs);
 DEBUG_rawFrames = options.raw;
 DEBUG_frameDetail = options.detail;
 
-let json;
-try {
-  json = require(`./${options.json}`);
-} catch (error) {
-  console.log('Error parsing JSON file:', options.json);
-  console.log(error);
-  process.exit();
-}
+if (options.readPacket) {
+  doPacket(options.readPacket, {read: true, label: 'Rcvd:'});
+} else if (options.writePacket) {
+  doPacket(options.readPacket, {read: false, label: 'Sent:'});
```

Comment:
Shouldn't this be `doPacket(options.writePacket, ...)`?

Reply:
Good catch.